clang/docs/ReleaseNotes.rst

```diff
@@ -72,6 +72,10 @@ sections with improvements to Clang's support for those languages.
 C++ Language Changes
 --------------------
 
+C++14 Feature Support
+^^^^^^^^^^^^^^^^^^^^^
+- Sized deallocation is enabled by default in C++14 onwards.
```

Comment:
It may be good to mention the flag enabled so that people met regression can disable it locally to workaround it.